Software/DumptruckUltra-MCU/Makefile

```diff
@@ -90,7 +90,7 @@ DISABLE_COMPONENTS=
 # by default, or otherwise not found by the build system.
 # TEAM03: Change this line to specify which file has the main you want to run
 # DEFAULT: Use main.cpp
-SOURCES=main.cpp
```

Comment:
Leave SOURCES unchanged for the main branch

Reply:
Resolved in 1e53ec0.